doc/source/parameters/cfd/initial_conditions.rst

```diff
@@ -0,0 +1,32 @@
+Initial Conditions
+-------------------
+It is often necessary to set-up complex initial condition when simulating transient problems. Furthermore, setting a coherent initial condition can greatly help the convergence of steady-state problem. Lethe offers three different strategies to set initial conditions : Nodal (through a function), L2 projection (through a function) and viscous. The viscous strategy solves the same CFD problem but with a user-defined viscosity. This can allow the user to initialize the velocity field with a Stokes-like flow for which convergence can be obtained more easily.
```

Comment:
```suggestion
It is often necessary to set-up complex initial conditions when simulating transient problems. Furthermore, setting a coherent initial condition can greatly help the convergence of steady-state problems. Lethe offers three different strategies to set initial conditions : Nodal (through a function), L2 projection (through a function) and viscous. The viscous strategy solves the same CFD problem but with a user-defined viscosity. This can allow the user to initialize the velocity field with a Stokes-like flow for which convergence can be obtained more easily.
```